src/main/resources/resource/framework/pom.xml.template

```diff
@@ -307,9 +307,16 @@
                 <configuration>
                   <filesets>
                     <fileset>
-                      <directory>.myrobotlab</directory>
+                      <directory>data/.myrobotlab</directory>
                       <followSymlinks>false</followSymlinks>
 				    </fileset>
+					<fileset>

```

Comment:
please use spaces, not tabs.

Reply:
This isn't / shouldn't be related to the program ab change.. is it ?

Reply:
agreed